apps/src/lib/levelbuilder/code-docs-editor/MethodToken.jsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import color from '@cdo/apps/util/color';
+
+export default function MethodToken({method, updateMethod}) {
```

Comment:
Nit: is the name `Token` common in this context? Assuming you're saving `MethodEditor` for the main method editing page, but maybe `MethodNameEditor` (would align with `FieldEditor`, etc.)?

Reply:
We do have components named `Token` but I was really struggling with the naming here. Like you said, I'm saving `MethodEditor` for the method edit page. I can use `MethodNameEditor`, though I might add a couple of small fields to it in the future.

Reply:
Is this going to be a child in the `MethodEditor`? Maybe `MethodSubEditor` or `MethodAttributeEditor` or like Ben suggested `MethodNameEditor` and rename if it actually does have more fields in the future

Reply:
It's just going to live in `ProgrammingClassEditor` for now. I renamed to `MethodNameEditor` for now -- the only other field I can think of maybe being here is the field to say a method is overloaded by another one (which doesn't exist in the DB yet so will be added in a follow-up)